dvc/utils/serialize/_yaml.py

```diff
@@ -14,14 +15,77 @@ def __init__(self, path):
         super().__init__(path, "YAML file structure is corrupted")
 
 
-def load_yaml(path, tree=None):
-    return _load_data(path, parser=parse_yaml, tree=tree)
+def recursive_render(tpl, values, max_passes=100):
```

Comment:
This is for nested interpolation, right?
```jinja
{{ var {{ var }} }}
```

Reply:
Yep. Not the most efficient thing, just trying to get something working. My team and I have been using this feature in our internal projects. It's been quite nice!